test/controllers/vaccines_controller_test.rb

```diff
@@ -2,6 +2,8 @@
 
 require 'test_case'
 
+# IMPORTANT NOTE ON CHANGES TO Time.now CALLS IN THIS FILE
+# Updated Time.now to Time.now.getlocal for Rails/TimeZone because Time.now defaulted to a zone. In this case it was the developer machine or CI/CD server zone.
```

Comment:
A little confused on this. This states the problem was that `Time.now` defaulted to the local time zone of the server, but won't `Time.now.getlocal` do the same?

Reply:
Yes you're correct. The issue that the cop has is that you need to be explicit about using a local time zone instead of implicit (which is what `Time.now` does)

Reply:
Oh ok, I guess I just misunderstood the comment.